app/models/assessment/training.rb

```diff
@@ -13,8 +13,9 @@ class Assessment::Training < ActiveRecord::Base
   attr_accessible :dependent_on, :dependent_on_attributes
 
   validates_presence_of :title, :exp, :open_at
+  validates_numericality_of :bonus_exp
```

Comment:
Prefer the new style validations `validates :column, numericality: value` over `validates_numericality_of`.